tests/example/planar/planar.dl

```diff
@@ -0,0 +1,79 @@
+// Determines whether a graph is planar i.e. has no lines that intersect
```

Comment:
Needs the standard license header. e.g. https://github.com/souffle-lang/souffle/blob/master/tests/semantic/load7/load7.dl